trieste/models/model_interfaces.py

```diff
@@ -88,24 +88,15 @@ def predict_y(self, query_points: TensorType) -> tuple[TensorType, TensorType]:
             f"Model {self!r} does not support predicting observations, just the latent function"
         )
 
-    def get_observation_noise(self) -> TensorType:
+    def get_observation_noise(self):
```

Comment:
why removing the output type?

Reply:
also, why not moving the method to the GPpredictor class? we should list here methods that we would expect all the models to have, no?